blocks/story-carousel-block/features/story-carousel/default.jsx

```diff
@@ -0,0 +1,217 @@
+
+	if (!elements.length) return null;
+
+	const getParentClassNames = () => {
+		if (elements && elements.length > 4) {
+			return `${BLOCK_CLASS_NAME} ${BLOCK_CLASS_NAME}__story-card-5-plus`;
+		}
+		return `${BLOCK_CLASS_NAME} ${BLOCK_CLASS_NAME}__story-card-4-less`;
+	};
+
+	return (
+		<Stack className={getParentClassNames()}>
+			{headline && headline.trim() !== "" && <Heading>{headline}</Heading>}
+
+			{elements && elements.length && (
+				<Carousel
+					id={identifier}
+					label={carouselLabel}
+					slidesToShow={slidesToShow}
+					nextButton={
+						<Carousel.Button id={identifier} label={rightArrowLabel}>
+							<Icon name="ArrowRight" />
+						</Carousel.Button>
+					}
+					previousButton={
+						<Carousel.Button id={identifier} label={leftArrowLabel}>
+							<Icon name="ArrowLeft" />
+						</Carousel.Button>
+					}
+				>
+					{elements.map((item, index) => {
+						const currentSlide = index + 1;
+						return (
+							<Carousel.Item key={item._id} label={`Slide ${currentSlide} of ${elements.length}`}>
+								<a className={`${BLOCK_CLASS_NAME}__story-card`} href={item.canonical_url}>
+									<Image alt={item.headlines.basic} src={testImage} />
+									{item.headlines?.basic ? (
+										<Heading
```

Comment:
You'll need to add a `<HeadingSection>` wrapper to ensure the heading here is incremented correctly